templates/server.erb

```diff
@@ -0,0 +1,10 @@
+server <%= @name %> {
+  bogus <%= @bogus ? 'yes' : 'no' %>;
+  edns <%= @edns ? 'yes' : 'no' %>;
+<%- if @key -%>
+  keys <%= @key %>;
```

Comment:
My mistake. I had actually gone to that page to read about the directive and misread it. I've just chekced the `named.conf` man page on my nameserver and it's actually much clearer about this syntax.